src/chart/generator/marker.cpp

```diff
@@ -166,8 +165,15 @@ void Marker::setIdOffset(size_t offset)
 
 std::string Marker::toJSON() const
 {
-	std::string res;
-	Conv::JSONObj{res}("categories",
+	static std::string res;
+	Conv::JSONObj jsonObj{res};
```

Comment:
Do not use static variable for JSONObj, or clear it manually before use.

Reply:
copypaste error, static is not needed

Reply:
```
std::string Marker::toJSON() const
{
	std::string res;
	{
		Conv::JSONObj jsonObj{res};
		appendToJSON(jsonObj);
	}
	return res;
}
```
Like this?